src/api/layers/KMLLayer.class.js

```diff
@@ -70,10 +70,31 @@ export default class KMLLayer extends AbstractLayer {
      * @returns {boolean}
      */
     isLegacy() {
-        if (this.metadata?.author === 'web-mapviewer') {
-            return false
+        return !!(this.metadata?.author !== 'web-mapviewer')
+    }
+
+    /**
+     * Clear the KML Admin ID
+     *
+     * Once the admin id has been cleared we cannot modify this KML anymore
+     */
+    clearAdminId() {
+        this.adminId = null
+        if (this.metadata) {
+            this.metadata.adminId = null
+        }
+    }
```

Comment:
This function seems to be called nowhere.

Reply:
True this is a left over, a first implementation was to get rid of the admin id when leaving the menu.